### ISSUE TITLE
Client crash after update to 1.6.2

### ISSUE THREAD

**Issue by cyraxan** [DArkHekRoMaNT/StoneQuarry#3]
after update to stonequarry@1.6.2 slabs are broken
all existed slabs crash client on any action, and new one from creative menu contains x0 stone.

```Running on 64 bit Windows with 16 GB RAM 
Game Version: v1.16.3 (Stable) 
Loaded Mods: advancedsmithing@1.1.0, weaponpackapatch@2.0.0, ancienttools@1.3.5, animalcages@2.0.6, axleinblocks@1.0.12, betterdrifters@2.1.3, dragonflies@1.0.3, necessaries@1.1.1, neededthings@1.0.0, nightwatcher@1.2.0, primitivesurvival@2.8.0, qptech@1.6.4, RangedWeapons@0.4.5, superfuel@2.0.7, locustmod@2.3.5, tinytweaks@1.1.0, tradeomat@1.6.0, viesblocks@2.0.2, viescraftmachines@1.5.2, game@1.16.3, alchemy@1.5.9, carrycapacity@0.6.4, fromgoldencombs@1.3.6-rc.4, helvehammerext@1.2.0, hudclock@2.6.0, lichen@1.6.0, medievalexpansion@3.7.1, morebags@1.0.0, petai@1.3.4, playercorpse@1.3.0, prospectorinfo@3.1.2, stonequarry@1.6.1, tpnet@1.5.1, usefulstuff@2.3.0, creative@1.16.3, survival@1.16.3, wildfarming@1.2.3, workbenchexpansion@1.4.1, xlib@0.6.8, xmelee@0.0.10, bullseye@2.2.4, cats@1.3.2, feverstonehorses@1.2.5, wildcraft@1.1.4, wolftaming@1.3.1, xskills@0.6.8
15.02.2022 21:14:22: Critical error occurred
System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в StoneQuarry.BlockRoughCutStorage.OnBlockBroken(IWorldAccessor world, BlockPos pos, IPlayer byPlayer, Single dropQuantityMultiplier)
   в Vintagestory.API.Common.CollectibleObject.OnBlockBrokenWith_Patch2(CollectibleObject this, IWorldAccessor world, Entity byEntity, ItemSlot itemslot, BlockSelection blockSel, Single dropQuantityMultiplier)
   в Vintagestory.Client.NoObf.ClientMain.OnPlayerTryDestroyBlock(BlockSelection blockSelection)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.ContinueBreakSurvival(BlockSelection blockSelection, Block block, Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   в Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   в Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   в _TcPKe0ld0EAbHR8YsMG1OLy9q8O._vg1QFi5yvtdWtcWdYGzwV9j9PVf(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._mSnyaJUIQhBb8A1j3UUn7uh5BBO(Single )
   в _SbWHHe8Ad6lBV8IRbvWFqut7ren._b365Cpnna1p8GSaDyyO3EH2VfOD(Single )
   в Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 476
   в OpenTK.GameWindow.DispatchRenderFrame() в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 452
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 375
   в _muFH2BhSjNKo8GSc2o33Nm6Ckup._SwLkq72vc0vcc3OnHXTAS19pm9p(_tVmXPdKJ3JBDWFc8tBtvgftcfhl , String[] )
   в _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )```

**Comment by DArkHekRoMaNT** [comment]
> new one from creative menu contains x0 stone

Creative slabs always contain 0 stone.  Use the special creative-only tool to add the neaded amount of stone

**Comment by cyraxan** [comment]
when I click right mouse button on creative slab with stonequarry:slabcontentsetter-quantity nothing happend, quantity of stones in slab not increasing.
If I clink with LMB client crash
survival  chopped slabs seems to work ok

```
Game Version: v1.16.3 (Stable) by Skif_97
Loaded Mods: advancedsmithing@1.1.0, weaponpackapatch@2.0.0, ancienttools@1.3.5, animalcages@2.0.6, axleinblocks@1.0.12, betterdrifters@2.1.3, dragonflies@1.0.3, emotemenu@1.0.3, necessaries@1.1.1, neededthings@1.0.0, nightwatcher@1.2.0, primitivesurvival@2.8.0, qptech@1.7.1, RangedWeapons@0.4.5, superfuel@2.0.7, locustmod@2.3.5, tinytweaks@1.1.0, tradeomat@1.6.0, viesblocks@2.0.2, viescraftmachines@1.6.0, game@1.16.3, zoombuttonreborn@1.2.0, alchemy@1.5.9, carrycapacity@0.6.4, extraoverlays@1.1.0, farmlife@1.2.2, fromgoldencombs@1.3.6-rc.4, gasapi@1.0.1, helvehammerext@1.2.0, hudclock@2.6.0, lichen@1.6.0, medievalexpansion@3.7.1, morebags@1.0.0, petai@1.3.5, playercorpse@1.3.0, prospectorinfo@3.1.2, stonequarry@1.6.2, tpnet@1.5.1, usefulstuff@2.3.1, creative@1.16.3, survival@1.16.3, workbenchexpansion@1.4.1, xlib@0.6.8, xmelee@0.0.10, campaigncartographer@2.1.0-pre.1, bullseye@2.2.4, cats@1.3.2, feverstonehorses@1.2.5, wildcraft@1.1.4, wolftaming@1.3.2, xskills@0.6.8
17.02.2022 11:32:06: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at StoneQuarry.BlockRoughCutStorage.OnBlockBroken(IWorldAccessor world, BlockPos pos, IPlayer byPlayer, Single dropQuantityMultiplier)
   at Vintagestory.API.Common.CollectibleObject.OnBlockBrokenWith_Patch2(CollectibleObject this, IWorldAccessor world, Entity byEntity, ItemSlot itemslot, BlockSelection blockSel, Single dropQuantityMultiplier)
   at Vintagestory.Client.NoObf.ClientMain.OnPlayerTryDestroyBlock(BlockSelection blockSelection)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleMouseInteractionsBlockSelected(Single dt)
   at Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnFinalizeFrame(Single dt)
   at Vintagestory.Client.NoObf.ClientEventManager.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.TriggerRenderStage(EnumRenderStage stage, Single dt)
   at Vintagestory.Client.NoObf.ClientMain.RenderToDefaultFramebuffer(Single dt)
   at _TcPKe0ld0EAbHR8YsMG1OLy9q8O._vg1QFi5yvtdWtcWdYGzwV9j9PVf(Single )
   at _SbWHHe8Ad6lBV8IRbvWFqut7ren._mSnyaJUIQhBb8A1j3UUn7uh5BBO(Single )
   at _SbWHHe8Ad6lBV8IRbvWFqut7ren._b365Cpnna1p8GSaDyyO3EH2VfOD(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 476
   at OpenTK.GameWindow.DispatchRenderFrame() in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 452
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) in C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:line 375
   at _muFH2BhSjNKo8GSc2o33Nm6Ckup._SwLkq72vc0vcc3OnHXTAS19pm9p(_tVmXPdKJ3JBDWFc8tBtvgftcfhl , String[] )
   at _XL1Xa21FKJVFetNmIumId5p6UDU._SwLkq72vc0vcc3OnHXTAS19pm9p(ThreadStart )
-------------------------------
```

**Comment by DArkHekRoMaNT** [comment]
The setter should be set to the desired amount, default is 0. You can see a help when you put an item in your hand (check if interaction help is enabled in the settings).

Crash issue should be fixed in 1.6.3
